src/components/ability/index.js

```diff
@@ -19,7 +22,12 @@ const Ability = ({ name, value, increment, decrement }) => {
                     />
                 </div>
                 <span className="ability-score__score-text">{value}</span>
-                <div className="ability-score__button ">
+                <div
+                    className="ability-score__button"
+                    style={{
+                        visibility: value === minValue ? 'hidden' : 'visible',
+                    }}
+                >
                     <Button
                         title=" "
                         icon="caret-left"
```

Comment:
I know this isn't actually part of this PR, but I think this qualifies for a refactor going forward. Passing in a blank title is nonsensical. Probably not in this PR, but I just wanted to point it out.